library/globals.inc.php

```diff
@@ -579,6 +579,13 @@ function gblTimeZones()
             xl('Are appointment category names to be translated?')
         ),
 
+        'npi_validate' => array(
+            xl('Validate NPI'),
+            'bool',                           // data type
+            '0',                              // default = false
+            xl('Check to enable NPI validation for Users and Address Book entries.')
```

Comment:
Recommend also noting that the validation is US-specific, just to be more clear.